crates/core/src/spk_client.rs

```diff
@@ -345,9 +323,11 @@ impl<I> SyncRequest<I> {
 #[must_use]
 #[derive(Debug)]
 pub struct SyncResult<A = ConfirmationBlockTime> {
-    /// The update to apply to the receiving [`TxGraph`].
-    pub graph_update: TxGraph<A>,
-    /// The update to apply to the receiving [`LocalChain`](crate::local_chain::LocalChain).
+    /// The update to apply to the receiving
+    /// [`TxGraph`](../../bdk_chain/tx_graph/struct.TxGraph.html).
+    pub graph_update: crate::tx_graph::Update<A>,
+    /// The update to apply to the receiving
+    /// [`LocalChain`](../../bdk_chain/local_chain/struct.LocalChain.html).
```

Comment:
Heh I guess this link is one way to do it but I think it's a mistake to mention `LocalChain` and `TxGraph` here. We are decoupling these things. What if we make a new chain type wouldn't a CheckPoint also be a valid update for it? (I'm pushing a commit).